api/lib/controllers/meeting.js

```diff
@@ -280,14 +297,70 @@ module.exports = {
         },
         { $unwind: '$meetings' },
         { $replaceRoot: { newRoot: '$meetings' } },
+        ...pipelineFilters,
```

Comment:
I like how you define the filters first then place them in here. It makes it a lot easier to reason about.